### PR TITLE
fix: node version in CI

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

**Comment by cehali** [comment]
Based on https://github.com/NomicFoundation/hardhat/issues/3877

File: .github/workflows/sdk-tests.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       run:
         working-directory: packages/sdk
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```
Comment (koonopek) [inline .github/workflows/sdk-tests.yml:17]: Missing `actions/setup-node`?

**Comment by koonopek** [comment]
So some specific node version was the issue, or the issue was that is wasn't specified?

**Review by koonopek — APPROVED** [review]
One minior question

File: .github/workflows/sdk-tests.yml (hunk 1 above)
Comment (cehali) [reply .github/workflows/sdk-tests.yml:17]: We need to specify it only in packages where we use hardhat

**Comment by cehali** [comment]
> So some specific node version was the issue, or the issue was that is wasn't specified?

It seems that 18.16 is breaking something but I assumed that when we use `actions/checkout@v2` we use an older version than 18 because we had warnings that we should switch to v3 to use a newer node.

**Review by hatskier — APPROVED** [review]

File: .github/workflows/sdk-tests.yml (hunk 1 above)
Comment (koonopek) [reply .github/workflows/sdk-tests.yml:17]: I think that this is good practice to specify it everywhere to be sure what node are we using

**Pull request merged**